### ISSUE TITLE
Distribution List allows ANNOUNCEMENT blank email address

### ISSUE THREAD

**Issue by jimimaseye** [hmailserver/hmailserver#183]
When setting up a distribution list, and selecting "_Announcements - Only allow messages from the following address_",  it allows to click SAVE and not actually enter an address (therefore leaving it blank).

Screen should check and enforce an entry in the address field if "announcement only address" option has been selected.

(I never further tested the consequences of leaving it blank but I would expect incorrect operation).

**Comment by martinknafve** [comment]
I won't fix this one, due to lack of time on my part.

**Comment by jimimaseye** [comment]
@RvdHout  Simple one to code for (a conditional check at time of entry)?  Want to take it on?

**Comment by RvdHout** [comment]
Where exactly is this happening, in webdmin or in GUI?

**Comment by jimimaseye** [comment]
It will be in Admin GUI (I dont use webadmin)

**Comment by Neustradamus** [comment]
@martinknafve: It is not good to close all unsolved tickets!

I have done all informations to change the status of hMailServer.

**Comment by martinknafve** [comment]
@Neustradamus Issues which there are no plans to address will be closed. If someone wants to pick it up and implement it then they can be re-opened. Please stop spamming unrelated messages into threads. If it happens again, I will block your account from further communication in this repo. Thanks.

**Comment by RvdHout** [comment]
@jimimaseye, there is (almost) no validation done anywhere withing the GUI, eg: enable spam option but leave/make score empty...all is possible, so to do that for just this item seems worthless to me

**Comment by jimimaseye** [comment]
> @jimimaseye, there is (almost) no validation done anywhere withing the GUI, eg: enable spam option but leave/make score empty, account forward without address...all is possible, so to do that for just this item seems worthless to me

Indeed, there isnt.  It's a tricky one:  in one sense you are right (why validate here when there isnt anywhere else) but on the other hand having missing validation does leave the software open to problem.  I guess the change could be seen as an improvement to solidify its functionality but I can also see that doing this should include checking the other areas that need it too.  Given that this particular feature is little used and therefore even less likely have people fall down the open trap door that the lack of validation gives, we could choose to leave it for now.

Maybe if the software development gets adopted by someone (in light of recent announcements by Martin) then perhaps the necessary change will be incorporated in a larger development program.

**Comment by Neustradamus** [comment]
@jimimaseye: Yes I have requested a better hMailServer development, he has started to change the github account to organization (not complete, description, logo, topics, etc. are not here), a good begining to have new developers.

But the easy solution to close unsolved tickets is not good because developers can not understand if it is solved or not.

I have requested to do announcement on ML, on the forum, on Twitter, on this GitHub to search new developers.

The hMailServer must be a secure products currently it is not perfect.

Ban one people on the forum because I have informed that SCRAM-SHA-*(-PLUS) are not supported and requested to add...
